homeassistant/components/dnsip/sensor.py

```diff
@@ -92,7 +104,11 @@ async def async_update(self) -> None:
             response = None
 
         if response:
-            self._attr_native_value = response[0].host
+            sorted_ips = sort_ips(
+                [res.host for res in response], querytype=self.querytype
+            )
+            self._attr_native_value = sorted_ips[0]
+            self._attr_extra_state_attributes["IPs"] = sorted_ips
```

Comment:
As mentioned on discord can we cap this if it's a crazy amount of hosts

Reply:
Added `MAX_RESULTS` and have set to 12.